qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/InactiveRouteStylingActivity.kt

```diff
@@ -211,8 +212,11 @@ class InactiveRouteStylingActivity : AppCompatActivity() {
         mapboxReplayer.play()
     }
 
-    private fun getRoute(): DirectionsRoute {
+    private fun getNavigationRoute(): NavigationRoute {
```

Comment:
```suggestion
    private val navigationRoute by lazy {
```
What about this instead? It would work as well without us having to decode options or hardcoding some fake UUID. They key is to use one instance of the route.

Reply:
The RouteOptions::enableRefresh needs to be false.  I found that without this there was a route refresh when navigation starts which gets a different route than the static one needed for consistent testing.  In order to use the updated options I need the the `fromJson` that takes an ID as an argument.  Since it's a static route it didn't seem like it was problem hardcoding an ID.  I can nullify it and make the parameter lazy.  The test was working so I didn't see a need to do anything further.